generators/app/tasks/fileCreators.js

```diff
@@ -0,0 +1,70 @@
+const generateRSScript = (command, options = '') => `rescripts ${command} ${options}`;
+
+const getPackageJsonAttributes = (projectName, projectVersion, repoUrl) => ({
+  name: projectName,
+  title: projectName,
+  version: projectVersion,
+  repository: {
+    type: 'git',
+    url: repoUrl
+  },
+  engines: {
+    node: '>= 10.15.3',
+    npm: '>= 6.9.0'
+  },
+  scripts: {
+    start: 'node ./scripts/start.js development',
+    'start-env': 'node ./scripts/start.js',
+    build: 'node ./scripts/build.js',
+    // eslint-disable-next-line no-extra-parens
+    ...(this.customized && { deploy: 'node ./scripts/deploy.js' }),
+    test: generateRSScript('test', '--env=jsdom'),
+    eject: './node_modules/react-scripts/bin/react-scripts.js eject',
+    lint: './node_modules/eslint/bin/eslint.js src',
+    'lint-fix': './node_modules/eslint/bin/eslint.js src --fix',
+    'lint-scss': "./node_modules/stylelint/bin/stylelint.js '**/*.scss' --fix",
```

Comment:
You overrode my changes to `lint-scss` and `lint-fix` in https://github.com/Wolox/react-bootstrap/pull/91/files